easybuild/easyconfigs/s/structure/structure-2.3.4-GCC-6.3.0.eb

```diff
@@ -0,0 +1,39 @@
+#
+# This file is an EasyBuild reciPy by Christian Meesters (2017)
+# HPC-group University of Mainz (https://hpc.uni-mainz.de)
+
+easyblock = 'MakeCp'
+
+name = 'structure'
+version = '2.3.4'
+
+homepage = 'https://web.stanford.edu/group/pritchardlab/structure.html'
+description = """The program structure is a free software package for using multi-locus
+ genotype data to investigate population structure. Its uses include infering 
+ the presence of distinct populations, assigning individuals to populations, 
+ studying hybrid zones, identifying migrants and admixed individuals, and estimating
+ population allele frequencies in situations where many individuals are migrants or admixed. 
+ It can be applied to most of the commonly-used genetic markers, including SNPS, 
+ microsatellites, RFLPs and AFLPs."""
+
+toolchain = {'name': 'GCC', 'version': '6.3.0'}
+
+source_urls = ['https://web.stanford.edu/group/pritchardlab/structure_software/release_versions/v%(version)s/']
+sources     = [{'download_filename': 'structure_kernel_source.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums   = ['f2b72b9189a514f53e921bbdc1aa3dbaca7ac34a8467af1f972c7e4fc9c0bb37']
+
+# use same binutils version that was used when building GCC toolchain
+builddependencies = [
+    ('binutils', '2.27', '', True)
+]
```

Comment:
@cmeesters This should go in the other easyconfig file, where you use `GCCcore` as toolchain. And this easyconfig file, that uses `GCC` should be removed.

Or, you should remove the easyconfig file using `GCCcore` as toolchain, and keep the one that uses `GCC` but then remove `binutils` as a build dep there...